conn/sasl.c

```diff
@@ -96,6 +96,13 @@ struct SaslSockData
   int (*close)(struct Connection *conn);
 };
 
+static char *sasl_authenticators[] =
+{
+  "ANONYMOUS", "CRAM-MD5", "DIGEST-MD5", "EXTERNAL", "G2", "GSSAPI",
+  "GSS-SPNEGO", "KERBEROS_V4", "LOGIN", "NTLM", "OTP", "PASSDSS", "PLAIN",
+  "SCRAM", "SRP"
+};
```

Comment:
I'd like the ARRAY APIs to be used instead of raw arrays whenever possible. See c24d17a99d16e81872f08f721a1793a57bbdaa92 for an example.

Reply:
nevermind, disregard this :)